file_system.go

```diff
@@ -321,6 +348,66 @@ func (f *ftpFile) Sys() interface{} {
 	return f.raw
 }
 
+var dirRegex = regexp.MustCompile(`^\s*(\S+\s+\S+\s{0,1}\S{2})\s+(\S+)\s+(.*)`)
+var dirTimeFormats = []string{
+        "01-02-06  03:04",
+        "01-02-06  03:04PM",
+        "2006-01-02  15:04",
+}
+
+// 08/03/2016  17:13    <DIR>          directory-name
+// 08/07/2016  17:20             1,135 file-name
+// or
+// 02-10-16  12:10PM           1067784 file.exe
+func parseDirLIST(entry string, loc *time.Location, skipSelfParent bool) (os.FileInfo, error) {
```

Comment:
220 Microsoft FTP Service
...
STAT
211-Microsoft FTP Service status:
     Logged in user: anonymous
     TYPE: ASCII; FORM: NONPRINT; STRUcture: FILE; transfer MODE: STREAM
     Data connection: none
211 End of status.
SYST
215 Windows_NT